packages/common/src/api/tan-query/events/utils.ts

```diff
@@ -0,0 +1,13 @@
+import { ID } from '~/models'
+
+import { QUERY_KEYS } from '../queryKeys'
+
+export const getEventQueryKey = (eventId: ID | null | undefined) => [
+  QUERY_KEYS.events,
+  eventId
+]
+
+export const getEventListQueryKey = ({ pageSize }: { pageSize?: number }) => [
+  QUERY_KEYS.events,
+  { pageSize }
+]
```

Comment:
Just a heads up, once I merge https://github.com/AudiusProject/audius-protocol/pull/11697, you can add typecheck support to your queryKeys in the query key type mappings file. This will let `get(set)QueryData` return typed data